lib/sync/installation.js

```diff
@@ -11,8 +11,21 @@ const tasks = {
 };
 const taskTypes = Object.keys(tasks);
 
+const updateNumberOfReposSynced = async (repos, subscription) => {
+  const repoIds = Object.keys(repos);
+
+  const syncedRepos = repoIds.filter((id) => {
+    // all 3 statuses need to be complete for a repo to be fully synced
+    const { pullStatus, branchStatus, commitStatus } = repos[id];
+    return pullStatus && branchStatus && commitStatus === 'complete';
```

Comment:
I think this will test if `pullStatus` is truthy, if `branchStatus` is truthy, and if `commitStatus` is exactly equal to `complete`. Maybe you'd prefer something like this?
```suggestion
    return pullStatus === 'complete' && branchStatus === 'complete' && commitStatus === 'complete';
```

Reply:
Good point will update.